registered/validate/validators.py

```diff
@@ -516,6 +516,30 @@ def validate_all_runs_have_blocks(rating):
         )
 
 
+def validate_all_runs_are_numeric(rating):
+    """
+    Validate each Run ID in each Piece in the CRW file does not have any 
+    alphabet characters.
+
+    When there are alphabet characters in the Run ID, operators are unable to log 
+    into that run in TM.
+    """
+
+    for piece in rating["crw"]:
+        if not isinstance(piece, parser.Piece):
+            continue
+
+        if all(not char.isalpha() for char in piece.run_id):
```

Comment:
```suggestion
        if piece.run_id.isdigit()
```
docs: https://docs.python.org/3.13/library/stdtypes.html#str.isdigit